### DISCUSSION TITLE
ili9488 tft 3.5" touch screen

### DISCUSSION THREAD

**Discussion by Shahijoon** [Bodmer/TFT_eSPI#2090] [Q&A - General] [answered]
Hi @Bodmer 
Would you please add a photo about how to connect my touchscreen(ili9488 3.5") pins to arduino UNO? Im using  voltage divider resistors to make 3.3v from 5v for logic pins, is it necessary to do the same for touch pins?

And which library should I include?

**Comment by Bodmer** [comment, marked as answer]
This library uses too much RAM and FLASH to be usable on an UNO. The Arduino forum is the best place to get advice.
